classes/Record.py

```diff
@@ -43,10 +44,17 @@ def edit_phone(self, old_phone, new_phone):
     def add_email(self, email):
         self.email = Email(email)
 
+    def add_birthday(self, birthday):
+        self.birthday = Birthday(birthday)
+        return "Birthday added"
+
     def __str__(self):
         result = f"Contact name: {self.name.value}, phones: {', '.join(p.value for p in self.phones)}"
 
         if self.email:
             result += f", email: {str(self.email)}"
 
+        if self.birthday:
```

Comment:
Try to enter command "all"
'Record' object has no attribute 'birthday'
Please rename
self.**birthday** = None

+ please remove data from address-book.pkl to avoid bugs => contacts with the wrong field **birthdays** may be saved there